src/constants/_constants/config.js

```diff
@@ -0,0 +1,46 @@
+
+// TODO: Configure travis to build a staging version
+// export const AWS_CONFIG = process.env.REACT_APP_STAGE === 'production'
+//     ? aws_config
+//     : aws_exports
+
+export const API_URL = process.env.REACT_APP_STAGE === 'production'
+  ? process.env.REACT_APP_PROD_API
+  : process.env.REACT_APP_STAGING_API
+
+export const CLIENT_URL = process.env.REACT_APP_STAGE === 'production'
+  ? 'https://app.ubcbiztech.com/'
+  : 'http://localhost:3000/'
```

Comment:
What's the difference between staging and dev here? Do we have a dev and a stg environment?

Reply:
Yup. `stage` [was introduced](https://github.com/ubc-biztech/bt-web/pull/188) to host our **MaxVP** while it is still in development
`prod` will now only host the **MinVP** to prevent users from being able to login and access MaxVP pages